iquip/apps/scheduler.py

```diff
@@ -0,0 +1,328 @@
+            self.argslayout.addWidget(self.name)
+            self.args = []
+            for key in info.arginfo:
+                self.args.append(QLabel(key + ': ' + str(info.arginfo[key])))
+                self.argslayout.addWidget(self.args[-1])
+            layout.addLayout(self.argslayout, 5)
+            self.editBtn = QPushButton("EDIT")
+            self.editBtn.clicked.connect(self.edit)
+            layout.addWidget(self.editBtn, 1)
+        else:
+            self.name = QLabel("None")
+            self.name.setFont(QFont("Arial", 10))
+            layout.addWidget(self.name)
+        self.setLayout(layout)
+
+    def changeInfo(self, info: ExperimentInfo):
+        """Change the information by modification.
+
+        Args:
+            info: The experiment information.
+        """
+        delete_items_of_layout(self.argslayout)
+        if info:
+            self.name.setText(info.name)
+            self.name.setFont(QFont("Arial", 15))
+            self.argslayout.addWidget(self.name)
+            self.args = []
+            for key in info.arginfo:
+                self.args.append(QLabel(key + ': ' + str(info.arginfo[key])))
+                self.argslayout.addWidget(self.args[-1])
+            self.editBtn = QPushButton("EDIT")
+            self.editBtn.clicked.connect(self.edit)
+
+    def data(self):
+        """Data transfer for displaying in ExperimentDelegate."""
+        return self.expInfo
+
+    def edit(self):
+        """Edition of experiment informations."""
+        # TODO(giwon2004): Display a frame for editing the values.
+        print("Button Clicked")
+
+
+class RunningExperimentView(QWidget):
+    """Widget for displaying the information the experiment, especially for the one running.
+    
+    Attributes:
+        argslayout: The HBoxLayout for displaying the experiment information besides its name.
+        name: The QLabel instance for displaying the experiment name.
+    """
+
+    def __init__(self, parent: Optional[QWidget] = None):
+        """Extended.
+        
+        Args:
+            info: The information of the experiment.
+        """
+        super().__init__(parent=parent)
+        layout = QVBoxLayout(self)
+        self.name = QLabel("None")
+        self.name.setFont(QFont("Arial", 10))
+        self.args = []
+        self.argslayout = QHBoxLayout()
+        layout.addWidget(self.name)
+        self.setLayout(layout)
+
+    def changeInfo(self, info: ExperimentInfo):
+        """Change the information by modification.
+
+        Args:
+            info: The experiment information.
+        """
+        delete_items_of_layout(self.argslayout)
+        if info:
+            self.name.setText(info.name)
+            self.name.setFont(QFont("Arial", 20))
+            self.args = []
+            for key in info.arginfo:
+                if key == "priority":
+                    continue
+                self.args.append(QLabel(key + ': ' + str(info.arginfo[key])))
```

Comment:
What is your purpose to store `QLabel` in both `args` and `argslayout`?
Is `args` used anywhere?

Reply:
It was for future work, maybe we can implement a widget by clicking on each of the arguments to edit it? But it is okay to go only with `argslayout` for now. I will edit it.